dietpi/dietpi-software

```diff
@@ -4332,6 +4339,189 @@ _EOF_
 			G_DIETPI-NOTIFY 0 'microblog.pub installation finished. In order to complete the setup, please run "microblog-pub configure" in a new bash session. Append the port ":8007" to the domain when being asked for it if you do not use a reverse proxy.'
 		fi
 
+		if To_Install 210 # lemmy
+        then
+            aDEPS=('make' 'gcc' 'libc6-dev' 'pkg-config' 'libssl-dev' 'protobuf-compiler' 'libprotobuf-dev' 'libpq-dev')
+			## Install up-to-date rustup (cargo version in debian is too old)
+			local lemm_user='lemmy'
+            local lemm_home="/home/$lemm_user"
+			# User
+            ##Create_User -G dialout,gpio,i2c -d "$lemm_home" "$lemm_user"
+            G_EXEC mkdir -p "$lemm_home"
+            Create_User -d "$lemm_home" "$lemm_user"
+            G_EXEC chown "$lemm_user":"$lemm_user" -R "$lemm_home"
+
+            ##G_EXEC sudo su -u $lemm_user mkdir -p $lemm_bin
+
+			## Install Rust, cargo
+			## We assume users will want to upload images, so we enable pictrs
+			G_DIETPI-NOTIFY 0 '[ INFO ] lemmy backend installation started'
+            G_EXEC curl -sSf 'https://sh.rustup.rs/' -o rustup-init.sh
+            G_EXEC chmod +x rustup-init.sh
+            G_EXEC_OUTPUT=1 G_EXEC ./rustup-init.sh -y --profile minimal
+            G_EXEC rm rustup-init.sh
+            export PATH="/root/.cargo/bin:$PATH"
+            G_EXEC_OUTPUT=1 G_EXEC cargo install lemmy_server --locked --features embed-pictrs
+			G_DIETPI-NOTIFY 0 '[ INFO ] lemmy backend installation finished'
+
+			## Install lemmy front-end server
+            ## Deps
+            local url='https://dl.yarnpkg.com/debian/pubkey.gpg'
+			G_CHECK_URL "$url"
+            G_EXEC eval "curl -sSfL '$url' | gpg --dearmor -o /etc/apt/trusted.gpg.d/dietpi-lemmy.gpg --yes"
+            G_EXEC eval "echo 'deb https://dl.yarnpkg.com/debian/ stable main' >  /etc/apt/sources.list.d/dietpi-yarn.list"
+            G_AGUP
+            G_AGI yarn nodejs
```

Comment:
This shouldn't be needed as you defined our up-to-date Node.js implementation as dependency already:
```suggestion
```
It includes `yarn`.